migrations/9_SCHEME-REP-Auction_Gen.js

```diff
@@ -1,20 +1,71 @@
-/* global artifacts */
+/* global artifacts, web3 */
 /* eslint no-undef: "error" */
+const DxAuction4Rep = artifacts.require('DxAuction4Rep')
+const DxAvatar = artifacts.require('DxAvatar')
+const DxController = artifacts.require('DxController')
+const GenToken = artifacts.require('GenToken')
+
+const dateUtil = require('../src/helpers/dateUtil')
+
+const registerScheme = require('./helpers/registerScheme')
+
+const {
+  reputationReward,
+  // TODO: update times, now in the past
+  lockingStartTime,
+  lockingEndTime,
+  numberOfAuctions,
+  wallet
+} = require('../src/config/schemes/old/auction4reputationparams.json')
```

Comment:
We should create out own config files, in a cleaner way than the ones created in `old` dir. 
Also the times are in the `conf/timePeriods` dir.

About the time period, I wrote the same in other PR, don't remember cause there's 15 opened! hahah
Anyways, pls remember that now that we have the date periods defined, we can give a better name to them (check the image I shared in the other PR)